MatrixSDK/Data/MXRoomSummary.m

```diff
@@ -314,6 +324,19 @@ - (void)eventDidChangeSentState:(NSNotification *)notif
     }
 }
 
+- (void)roomDidFlushData:(NSNotification *)notif
+{
+    MXRoom *room = notif.object;
+    if (_mxSession == room.mxSession && [_roomId isEqualToString:room.state.roomId])
+    {
+        NSLog(@"[MXRoomSummary] roomDidFlushData: %@", _roomId);
```

Comment:
Why don't you use the existing `resetRoomStateData` method?

Reply:
Idk. It may be better.